tests/unit_tests/dagmc/test_plot.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import openmc
+
+
+pytestmark = pytest.mark.skipif(
+    not openmc.lib._dagmc_enabled(), reason="DAGMC CAD geometry is not enabled."
+)
+
+def test_ploting_dagmc_geometry():
+    """Test plotting a DAGMC geometry with OpenMC. This is different to CSG
+    geometry plotting as the path to the DAGMC file needs handling."""
+
+    with openmc.config.patch('resolve_paths', True):
+        dag_universe = openmc.DAGMCUniverse(filename='dagmc.h5m')
```

Comment:
This will only work if the test is executed from the `tests/unit_tests/dagmc` directory. To work consistently it needs to reference the test directory path as in `request.path.parent / 'dagmc.h5m'` where `request` is a special pytest fixture.